src/requirements/requirement-exam-mapping.ts

```diff
@@ -0,0 +1,122 @@
+import examData, {
```

Comment:
I could! Although I think it might be better to keep complicated logic out of the `data/` directory. i.e. if we want to update the AP/IB exam data, we'd go to `data/ExamCredit.ts`; if we want to update the mappings logic, we'd go to this file.